homeassistant/components/philips_air_purifier/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""The Philips Air Purifier integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .client import ReliableClient
+from .const import COAP_PORT, DOMAIN
+
+PLATFORMS: list[str] = ["fan"]
```

Comment:
```suggestion
PLATFORMS: list[str] = [Platform.FAN]
```

+ `from homeassistant.const import Platform`